introspection-engine/introspection-engine-tests/tests/identify_version/mod.rs

```diff
@@ -340,3 +340,21 @@ async fn introspect_mssql_non_prisma_empty(api: &TestApi) -> TestResult {
 
     Ok(())
 }
+
+// TiDB
+
+#[test_connector(tags(TiDB))]
+async fn introspect_tidb_non_prisma(api: &TestApi) -> TestResult {
```

Comment:
Does this test also pass for MySQL? If so, we might just be able to replace the one above with this test (effectively just replace the type up there).

Reply:
Yes, merge them in https://github.com/prisma/prisma-engines/pull/3308/commits/c4caaf85f6e440cf4a39083ccc6ac13cc250aef5